packages/runtime/src/utils/load.ts

```diff
@@ -135,7 +135,12 @@ export async function getRemoteEntry({
       });
     }
   }
-  return globalLoading[uniqueKey];
+
+  return (globalLoading[uniqueKey] as Promise<RemoteEntryExports>).finally(
```

Comment:
Why do you need to remove this when removing it will result in repeated loading in the micro-front end scenario